src/main/java/mods/railcraft/common/plugins/forge/EnergyPlugin.java

```diff
@@ -78,7 +78,7 @@ public static int pushToTiles(TileEntity te, AdjacentTileCache tileCache, int pu
             if (source != null && canTileReceivePower(tile, side.getOpposite())) {
                 IEnergyStorage receiver = tile.getCapability(ENERGY, side.getOpposite());
                 if (receiver != null) {
-                    int amountToPush = source.extractEnergy(pushPerSide, false);
+                    int amountToPush = source.extractEnergy(pushPerSide, true);
                     if (amountToPush > 0) {
                         int amountPushed = receiver.receiveEnergy(amountToPush, false);
```

Comment:
Does this one need change, too?

Reply:
No, the receiving shouldn't be simulated.

Reply:
Ah, true means simulating. This is indeed inconvenient